tests/contrib/anthropic/utils.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+import vcr
+
+
+def iswrapped(obj):
+    return hasattr(obj, "__dd_wrapped__")
+
+
+# VCR is used to capture and store network requests made to Anthropic.
+# This is done to avoid making real calls to the API which could introduce
+# flakiness and cost.
+
+
+# To (re)-generate the cassettes: pass a real Anthropic API key with
+# ANTHROPIC_API_KEY, delete the old cassettes and re-run the tests.
+# NOTE: be sure to check that the generated cassettes don't contain your
+#       API key. Keys should be redacted by the filter_headers option below.
+# NOTE: that different cassettes have to be used between sync and async
+#       due to this issue: https://github.com/kevin1024/vcrpy/issues/463
+#       between cassettes generated for requests and aiohttp.
+def get_anthropic_vcr(subdirectory_name=""):
```

Comment:
```suggestion
def get_request_vcr():
```
Not required to have subdirectory name as we don't have to cover different minor versions of anthropic (which required different sets of cassette files)